### DISCUSSION TITLE
Why my middleware cannot get authorization token

### DISCUSSION THREAD

**Discussion by hiimwillow** [vercel/next.js#66570] [App Router]
Please guys need help been stuck in this for few days
my middleware doesnt capture auth token it gives me a null.
If i couldnt find the answer im going back to vitejs with nodejs backend

![image](https://github.com/vercel/next.js/assets/61561641/9fe2162b-af4f-4b13-8f2c-c9ea0f98f45e)
![image](https://github.com/vercel/next.js/assets/61561641/2b6ec084-ac1e-4fd0-b753-5943f1e6e56b)
![image](https://github.com/vercel/next.js/assets/61561641/766a1224-af8f-411f-b596-1e9f3404e10b)

**Comment by SajeeTechi** [comment]
@hiimwillow what was the reason behind this issue?
